charts/govuk-rails-app/templates/service.yaml

```diff
@@ -13,7 +13,11 @@ metadata:
 spec:
   type: {{ .Values.service.type }}
   ports:
-    - port: {{ .Values.service.port }}
+    - name: app
+      port: {{ .Values.service.port }}
       targetPort: {{ .Values.nginxPort }}
+    - name: monitor
```

Comment:
Good to use the names consistently (metrics not monitor)

Reply:
yup agreed, will get that changed